addons/udes_stock/tests/common.py

```diff
@@ -391,6 +391,23 @@ def create_move(cls, pickings, products_info, **kwargs):
         moves = Picking._create_move(move_values)
         return moves
 
+    @classmethod
+    def create_single_move(cls, product, qty, picking, **kwargs):
```

Comment:
Q: Why do we do this when create_move exists?

Reply:
Can we not just use create_move, which uses the functionality emplyed by the code outside of tests?

Reply:
fixed